src/metrics/test-coverage/index.ts

```diff
@@ -10,9 +10,12 @@ import {
 import { WorkflowStepCompleted } from "@octokit/webhooks-types";
 import { getTimestamp } from "../../shared/get-timestamp";
 import { isNameAboutTest } from "./is-name-about-test";
+import { Octokit } from "@octokit/rest";
 
 export const collectWorkflowsTestCoverageMetrics = async (
-  triggerEvent: SignedTriggerEvent
+  triggerEvent: SignedTriggerEvent,
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  repoClient: Octokit
```

Comment:
instead of having the unused war `repoClient` in multiple metrics we could make the `repoClient` optional in the interface definition.

In that case we also need to make sure that it is treated as possible `undefined` in the metrics where it is needed, e.g. `documentation-updated`.

I will leave comments in example places where changes would need to be made.

Reply:
If found a solution: `repoClient` is optional, and functions which do not use the parameter define this parameter with a leading underscore. In addition, the unused-vars error is disabled for this kind of parameter names.